assets/src/stories-editor/plugins/export.js

```diff
@@ -56,21 +78,19 @@ const handleExport = ( { postId, createErrorNotice, createSuccessNotice } ) => {
 							a.download = matches[ 1 ];
 							a.click();
 						} else {
-							createErrorNotice( errorMsg, {
-								id: 'amp-story-export__error-notice',
-							} );
+							showErrorNotice();
 						}
-					} );
+					} )
+					.catch( showErrorNotice );
 			} else {
 				// Handle the returned JSON error.
 				response.json()
 					.then( ( error ) => {
-						createErrorNotice( ( error.data && error.data.errorMessage ) ? error.data.errorMessage : errorMsg, {
-							id: 'amp-story-export__error-notice',
-						} );
-					} );
+						showErrorNotice( ( error.data && error.data.errorMessage ) ? new Error( error.data.errorMessage ) : null );
+					} )
+					.catch( showErrorNotice );
```

Comment:
Will writing it this way pass the error message to the function as a param automatically?

Reply:
Yes. The callback is passed an `Error` object which then has a `message` property.